spec/models/booking_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "rails_helper"
+
+RSpec.describe Booking, type: :model do
+
+  describe "callback" do
+    it "clear reason if not rejected" do
+      booking = FactoryBot.create(:booking)
+      booking.update status: :confirmed
+      expect(booking.reason).to be nil
+    end
+  end
+
+  describe "instance method" do
+    let(:booking) {FactoryBot.create(:booking)}
```

Comment:
Tại sao chỗ này dùng let mà ko dùng let! em

Reply:
em định cho mỗi it là 1 booking riêng, tại khi update status, thì có cái status rejected sẽ ràng buộc thêm phải có reason. nên em ko để là dùng chung 1 booking